### PR TITLE
Fixing Minio newest package version incompability

### PR DESCRIPTION
**Pull request by Synicix**
For now I changed to except all errors to make it work. I need to dig more into how minio does error handling to shrink the error scope to be more specific.

### CODE REVIEW

**Comment by guzman-raphael** [comment]
@Synicix Looks like from [here](https://github.com/minio/minio-py/blob/beb3d531bee8ed3b778ee673ce296f1f4e5bc2dd/minio/error.py#L67) you can check `minio.error.S3Error._code`. It appears that the code translation is done based on [this](https://github.com/minio/minio-py/blob/beb3d531bee8ed3b778ee673ce296f1f4e5bc2dd/minio/api.py#L329-L360) mapping.

File: datajoint/s3.py [hunk 1]
```diff
@@ -59,17 +62,23 @@ def exists(self, name):
...
             return self.client.stat_object(self.bucket, str(name)).size
-        except minio.error.NoSuchKey:
-            raise errors.MissingExternalFile from None
-
+        except minio.error.S3Error as e:
+            if e.code == 'NoSuchKey':
+                raise errors.MissingExternalFile from None
+            else:
+                raise e
+
```
Comment (guzman-raphael) [inline datajoint/s3.py:81]: Just took a quick look and looks like there is a simple whitespace issue.
```suggestion

```
To run the syntax tests locally, see [here](https://github.com/datajoint/datajoint-python/blob/143ba7b55fd292a0737e72d058670bcb5c6db064/local-docker-compose.yml#L89). To run the style tests locally, see [here](https://github.com/datajoint/datajoint-python/blob/143ba7b55fd292a0737e72d058670bcb5c6db064/local-docker-compose.yml#L90). We should probably add a comment/note in `local-docker-compose.yml` to be more clear about this.

**Review by guzman-raphael — CHANGES_REQUESTED** [review]

File: datajoint/s3.py (hunk 1 above)
Comment (Synicix) [reply datajoint/s3.py:81]: Sigh I thought I removed it all....

**Review by dimitri-yatsenko — APPROVED** [review]

**Review by guzman-raphael — APPROVED** [review]
Looks good! :rocket:

**Pull request merged**